src/neovim.rs

```diff
@@ -138,18 +140,57 @@ pub fn from_value(config_value: &Value) -> Option<LsConfig> {
                 .map(|s| String::from(s))
                 .collect::<Vec<String>>();
             root_markers = Some(data);
-        }
+        } else if k == "indentation" {
+            indentation = v.as_u64()?;
+        } else if k == "indentation_with_space" {
+			indentation_with_space = v.as_bool()?;
+		}
     }
     if let (Some(root_markers), Some(command)) = (root_markers, command) {
         Some(LsConfig {
             root_markers,
             command,
+            indentation,
+			indentation_with_space
```

Comment:
miss formatted here, did you run formatting :joy:

Reply:
:joy: working on formatting feature and forgot to format :rofl:

Reply:
Fixed :+1: